packages/create-aragon-app/src/commands/create-aragon-app.js

```diff
@@ -180,16 +154,18 @@ exports.handler = async function({
   )
 
   return tasks.run().then(() => {
-    reporter.success(`Created new application ${name} in path ./${basename}/
-
-Start your Aragon app by typing:
+    reporter.success(`Created new application ${name} in path ./${basename}/\n`)
 
-  cd ${basename}
-  npm start
+    if (template === 'react')
```

Comment:
Why do we only print the info for the buidler boilerplate?

Reply:
Because in the case of the tutorial we don't want to print it. And in case of the CLI, the steps are more complex than that 😅

Reply:
Ah indeed it makes sense not to display it for the tutorial hehe